bindings/python/test/test_mongocrypt.py

```diff
@@ -0,0 +1,224 @@
+# Copyright 2019-present MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test the mongocrypt module."""
+
+import os
+import sys
+
+from bson import json_util, BSON
+from bson.binary import STANDARD
+from bson.codec_options import CodecOptions
+
+sys.path[0:0] = [""]
+
+from pymongocrypt.binding import lib
+from pymongocrypt.errors import MongoCryptError
+from pymongocrypt.mongocrypt import (MongoCrypt,
+                                     MongoCryptBinaryIn,
+                                     MongoCryptBinaryOut,
+                                     MongoCryptOptions)
+
+from test import enable_faulthandler, unittest
+
+
+enable_faulthandler()
+
+
+# Data for testing libbmongocrypt binding.
+DATA_DIR = os.path.realpath(os.path.join(os.path.dirname(__file__), 'data'))
+
+
+class TestMongoCryptBinary(unittest.TestCase):
+
+    def test_mongocrypt_binary_in(self):
+        with MongoCryptBinaryIn(b'1\x0023') as binary:
+            self.assertEqual(binary.to_bytes(), b'1\x0023')
+        self.assertIsNone(binary.bin)
+
+        with MongoCryptBinaryIn(b'') as binary:
+            self.assertEqual(binary.to_bytes(), b'')
+        self.assertIsNone(binary.bin)
+
+        binary = MongoCryptBinaryIn(b'123')
```

Comment:
Can you clarify what is being tested here with the creation of `binary`? Is it just the use of this type outside of a context manager?